dcpy/test/lifecycle/ingest/test_transform.py

```diff
@@ -1,103 +1,69 @@
-from datetime import date
 import geopandas as gpd
 import pandas as pd
 import pytest
-from typing import cast
 import yaml
+from pydantic import TypeAdapter, BaseModel
 
-from dcpy.models import file
-from dcpy.models.lifecycle.ingest import Template
-from dcpy.models.lifecycle.ingest import LocalFileSource, Config, FunctionCall
+from dcpy.models.file import Format
+from dcpy.models.lifecycle.ingest import FunctionCall
 from dcpy.lifecycle.ingest import (
-    configure,
     transform,
     PARQUET_PATH,
 )
 
 from dcpy.utils import data
-from . import RESOURCES, TEST_DATA_DIR, FAKE_VERSION
+from . import RESOURCES, TEST_DATA_DIR
+
+
+class TestConfig(BaseModel):
```

Comment:
I created this pydantic class to validate objects from `transform_to_parquet_template.yml`. I expect this yaml file to grow with more test cases, and it would be nice to validate them. I'm not sure about the location of this class, however. Should it be moved to `models` module or stay here?

Reply:
Love this! I would keep it here for sure.